keras-dreambooth-sprint/README.md

```diff
@@ -99,28 +139,29 @@ output = gr.Gallery(label="Outputs").style(grid=(2,2))
 # pass function, input type for prompt, the output for multiple images
 gr.Interface(infer, inputs=["text"], outputs=[output]).launch()
 ```
+Q: should the part above be in the Collab already so it's end-to-end?
```

Comment:
no, we already give link to Space and application file is over there with more features.

Reply:
The expectation of this section is that a user will locally do an `app.py` and then copy paste/drag and drop it to the Space. I was wondering if it does not make sense to have a small gradio demo at the end of the colab so people learn to run a demo in colab, and then copy paste it to Spaces